docs/0-Preparing/continuing_side_projects.md

```diff
@@ -0,0 +1,53 @@
+## Continuing side projects
+
+This section continues the discussion on side projects. It's meant to go in a little more depth if you already have some coding and side project experience, but it might still be an interesting read otherwise.
+
+### More signals that side projects can give the employer
+
+**You can work in a team**
+
+- *Example*: You make a platformer game with 2 friends and publish it online. It only has one hour of gameplay but it works great!
```

Comment:
I guess that's true depending on the game (an RPG requires 1h of arts/dialogue/etc, flappy bird can be played for 10h with very basic mechanisms). I'll put "there's only one level in the game" instead.